### PR TITLE
PostgreSQL & DB installation cleanup

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/5778
- Converted linear database installation steps to tabbed content approach
- Attempted to remove legacy PostgreSQL details
- Cleaned up syntax and formatting to simplify content display

Goal: Provide generic DB installation steps that are no longer version-specific.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@agnivade & @nab-77 - We're moving towards delivering more generic installation and setup instructions that reduce maintenance overhead when new component releases become available. 

Can I get your help to identify any leftover documentation within the scope of this PR that's specific to a particularly database version, please (MySQL or PostgreSQL)? I suspect that some of the commands we include (as examples or otherwise) may have been developed with specific db versions available at the time.

**Comment by agnivade** [comment]
@cwarnermm - Is there any set of pages/commands to look at?

**Review by agnivade — APPROVED** [review]

**Comment by cwarnermm** [comment]
@agnivade - You can focus reviews on the commands within the following 3 files in this PR:
- installing-ubuntu-2004-LTS.rst
- install-rhel-8.rst
- install-debian.rst

And thank you! I suspect that we still have some older legacy 9.4 examples present.

File: source/install/installing-ubuntu-2004-LTS.rst [hunk 1]
```diff
@@ -20,18 +20,118 @@ Install a production-ready Mattermost system on up to three machines.
...
+
+  .. tab:: Install MySQL
+    
+    Install and set up a MySQL database for use by the Mattermost server.
+
+    1. Log into the server that will host the database, and open a terminal window.
+
+    2. Install MySQL.
+
+    3. Run ``sudo mysql_secure_installation`` and follow the instructions.
```
Comment (agnivade) [inline source/install/installing-ubuntu-2004-LTS.rst:46]: What is this command?

File: source/install/install-rhel-8.rst [hunk 2]
```diff
@@ -20,18 +20,146 @@ You can also use these instructions to install Mattermost on CentOS 8 or Oracle
...
+
+    4. Set PostgreSQL to start on boot by running ``sudo systemctl enable postgresql``.
+
+    5. Start the PostgreSQL server by running ``sudo systemctl start postgresql``.
+
+    6. Switch to the *postgres* Linux user account that was created during the installation by running ``sudo -iu postgres``.
+
+    7. Start the PostgreSQL interactive terminal by running ``psql``.
+
+    8.  Create the Mattermost database by running ``postgres=# CREATE DATABASE mattermost WITH ENCODING 'UTF8' LC_COLLATE='en_US.UTF-8' LC_CTYPE='en_US.UTF-8' TEMPLATE=template0;``.
```
Comment (agnivade) [inline source/install/install-rhel-8.rst:100]: Any reason this is different from the Ubuntu command? Ubuntu is just `CREATE DATABASE mattermost`

**Review by agnivade — COMMENTED** [review]
I don't have experience with Debian or RHEL so something might get missed out. But left some comments.

File: source/install/install-rhel-8.rst (hunk 2 above)
Comment (cwarnermm) [reply source/install/install-rhel-8.rst:100]: @nab-77 - Any idea why we have such an elaborate command documented?

File: source/install/installing-ubuntu-2004-LTS.rst (hunk 1 above)
Comment (cwarnermm) [reply source/install/installing-ubuntu-2004-LTS.rst:46]: @nab-77 - Any ideas?

Comment (nab-77) [reply source/install/installing-ubuntu-2004-LTS.rst:46]: Its a shell script to help admins secure their mysql install. Not sure we need to carry this level of detail here. Just need to ensure we call out the user and access rights we need.

File: source/install/install-rhel-8.rst (hunk 2 above)
Comment (nab-77) [reply source/install/install-rhel-8.rst:100]: I would need to look into the defaults variance between Ubuntu and RHEL, but right now I don't have the bandwith. Immediate thoughts are:

1. Its been in the docs for some time and so its not causing an issue right now.
2. We don't really need to provide this level of detail here. Simple the DB and any specific tuning/config for said DB.

@cwarnermm the ongoing review will remove any redundant information. For 7.2 I see no impact on leaving this as is.

**Pull request merged**